src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -153,6 +156,27 @@ public void onSpoutOpen(
 
         createSidelineTriggers();
 
+        final long refreshIntervalSeconds;
+
+        // This should be set by SidelineConfig.setDefaults(), but just in case we want to avoid the NullPointer
+        if (spoutConfig.containsKey(SidelineConfig.REFRESH_INTERVAL_SECONDS)) {
+            refreshIntervalSeconds = ((Number) spoutConfig.get(SidelineConfig.REFRESH_INTERVAL_SECONDS)).longValue();
+        } else {
+            refreshIntervalSeconds = 600;
+        }
+
+        final long refreshIntervalMillis = TimeUnit.SECONDS.toMillis(refreshIntervalSeconds);
+
+        // Repeat our sidelines check periodically
+        final Timer timer = new Timer();
+        timer.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                loadSidelines();
+            }
+        }, refreshIntervalMillis, refreshIntervalMillis);
+
+        // Why not just start the timer at 0? Because we want to block onSpoutOpen() until the first run of loadSidelines()
         loadSidelines();
```

Comment:
I think you want to run loadSidelines before you schedule the task?  You also may want to make loadSidelines() synchronized to prevent multiple threads/tasks from running it at the same time?